app/src/main/java/org/greenstand/android/TreeTracker/userselect/UserSelectScreen.kt

```diff
@@ -71,22 +81,91 @@ fun UserSelectScreen(
             )
         }
     ) {
+        LazyVerticalGrid(
+            cells = GridCells.Fixed(2),
+            modifier = Modifier.padding(it),  // Padding for bottom bar.
+            contentPadding = PaddingValues(start = 8.dp, end = 8.dp, top = 10.dp)
+        ) {
+            items(state.planters) { user ->
+                UserButton(
+                    user = user,
+                    state = state,
+                ) { viewModel.selectUser(user) }
+            }
+        }
+    }
+}
 
+@Composable
+fun UserButton(
+    user: PlanterInfoEntity,
+    state: UserSelectState,
```

Comment:
We're only passing in the screens state object to know if it's selected. To decouple the list item from the screen I'd suggest passing in an isSelected boolean instead.